packages/amplication-client/src/CommandPalette/CommandPalette.tsx

```diff
@@ -33,13 +43,43 @@ export class NavigationCommand implements Command {
   constructor(
     private readonly history: History,
     public readonly name: string,
-    public readonly link: string
+    public readonly link: string,
+    public readonly type: string,
+    public readonly isCurrentApp: boolean,
+    public readonly showAppData: boolean,
+    public readonly appName?: string,
+    public readonly appColor?: string
   ) {}
   command() {
     this.history.push(this.link);
   }
 }
 
+function RenderCommand(suggestion: Command) {
+  // A suggestion object will be passed to your custom component for each command
+  const { appColor, appName, name, highlight, showAppData, type } = suggestion;
+  return (
+    <>
+      {showAppData && (
+        <>
+          <ApplicationIcon name={appName || ""} color={appColor} />
+          <span className="command-palette__app-name">{appName}</span>
+        </>
+      )}
+      <Icon icon={type} />
+      {highlight && highlight[0] ? (
+        <span dangerouslySetInnerHTML={{ __html: highlight[0] }} />
```

Comment:
What if you have multiple highlights?

Reply:
The first highlight is for the "name" property, the second one is for the appName. I thought there is no benefit from highlighting the app name. 
I will have a session with @Leeyaacov tomorrow and if needed, I will add it.